src/basic/path-util.c

```diff
@@ -202,9 +202,9 @@ int path_make_relative(const char *from, const char *to, char **ret) {
 }
 
 char* path_startswith_strv(const char *p, char **set) {
-        char **s, *t;
-
         STRV_FOREACH(s, set) {
+                char *t;
```

Comment:
Doesn't really matter, but as I understand it, the iterator is now safe to modify, so this t could be done away with…

Reply:
the function doesn't modify the iterator though?

Reply:
I was implying that we can reuse s instead of declaring t.

Reply:
Actually, here `t` can be dropped and the iterator can be reused. But I prefer the current form for readability.

Reply:
Yeah, it's better to keep it.